### ISSUE TITLE
Write protection?

### ISSUE THREAD

**Issue by andreaskoerber** [bailey27/cppcryptfs#68]
I just installed cppcryptfs on Win10.
Creating and mounting an FS was ok. First copies of files into it, too. They are visible and usable from another machine with the same setup.
The next copies of files yielded erors, though, stating I had to remove the write protection.

How did this come along and how do I fix it?
Thx in adv
A

**Comment by andreaskoerber** [comment]
Could it be the case that write protection is triggered once two machines are accessing the same (network) volume simultaneously?

**Comment by bailey27** [comment]
Today I made an encrypted FS on one Windows 10 64-bit machine in a directory that was shared.  I copied stuff into it.  Then I mounted the shared folder on another Windows 10 64-bit machine and mounted the encrypted FS.  I was then able to copy more stuff into the encrypted FS from both machines without any write protection problems.  

I shared the drive to myself with full ownership.  Maybe you have a permissions problem with the network share?
